build/llamafile-convert

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+FILE=$1
+TYPE=${2:-both}
+SCRIPTNAME=$(basename $0)
+
+if [ -z "$FILE" ]; then
+  echo "Usage: $SCRIPTNAME <gguf file or url> [cli|server|both]"
+  exit 1
+fi
+
+# if the file starts with http
+if [[ $FILE == http* ]]; then
+  # download the file
+  # if the filename contains ?download=true, remove it
+  FILE=$(echo $FILE | sed 's/?download=true//g')
+  # get the filename
+  FILENAME=$(echo $FILE | sed 's/.*\///g')
+  echo "Downloading $FILENAME"
+  wget -O $FILENAME $FILE
+  # get the filename
+  FILE=$FILENAME
+fi
+
+# replace .gguf with .llamafile
+LLAMAFILE_NAME=$(echo $FILE | sed 's/.gguf/.llamafile/g')
+# replace .gguf with -server.llamafile
+LLAMAFILE_SERVER_NAME=$(echo $FILE | sed 's/.gguf/-server.llamafile/g')
+LLAMAFILE_PATH=$(which llamafile)
+LLAMAFILE_SERVER_PATH=$(which llamafile-server)
+CLI_ARGS=$(cat <<EOF
```

Comment:
You can do this using builtins as follows:

```bash
CLI_ARGS="-m
$FILE
...
"
```

Then, rather than echo'ing it later on, do this:

```sh
printf %s "$CLI_ARGS" >.args
```